csharp/test/Ice/interceptor/AllTests.cs

```diff
@@ -170,6 +170,7 @@ public static IMyObjectPrx Run(TestHelper helper)
 
             {
                 // throwing from an interceptor stops the interceptor chain
+                // TODO: this breaks colocation as the new communicator does not contain any OAs
```

Comment:
It's not clear what this comment is about and how it is related to the PR.

Reply:
This test was failing with `ssl` because creating a new communicator brakes colocation and we were are not copying the ssl configuration into the new communicator. @pepone disable colocation for this test, so I'll just remove the comment.